frontend/test/unit/specs/components/VTag/v-tag.spec.js

```diff
@@ -0,0 +1,50 @@
+import { screen } from "@testing-library/vue"
+
+import { render } from "~~/test/unit/test-utils/render"
+
+import VTag from "~/components/VTag/VTag.vue"
+
+describe("VTag", () => {
+  let props = null
+  let options = null
+
+  beforeEach(() => {
+    props = {}
+    options = { propsData: props }
+  })
+
+  it("should render an anchor tag by default", () => {
```

Comment:
Something like this might be more useful, i haven't tested though:

```ts
options.slots = {
  default: "Hello world",
}

const link = screen.getByRole('link', { name: "Hello world" })
expect(link).toHaveAttribute('href', 'https://example.com/')
```

This seems like the primary case we want to test, that a link is rendered with the correct text and href.